src/pages/Contact.jsx

```diff
@@ -1,11 +1,144 @@
+      newErrors.message = "Message must not exceed 250 characters.";
+    }
+
+    return newErrors;
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const validationErrors = validate();
+    if (Object.keys(validationErrors).length > 0) {
+      setErrors(validationErrors);
+    } else {
+      setErrors({});
+      setSuccessMessage("Reservation completed successfully!");
+      setFormData({ name: "", people: "", message: "" });
+      setSelectedDateTime(null);
+    }
+  };
+
+  return (
+    <div className="contact-page">
+      <h1 className="contact-title">Contact</h1>
+      <p className="contact-description">
+        Find us at some address at some place or call us at 05050515-122330
+      </p>
+      <p className="contact-note">
+        <strong>FYI!</strong> We offer full-service catering for any event, large or small. We
+        understand your needs and we will cater the food to satisfy the biggest criteria of them
+        all, both look and taste.
+      </p>
+      <h2 className="form-heading">
+        Reserve a table, ask for today’s special or just send us a message:
+      </h2>
+      {successMessage && <p className="success-text">{successMessage}</p>}
+      <form className="contact-form" onSubmit={handleSubmit}>
+        <input
+          type="text"
+          name="name"
+          className={`form-input ${errors.name ? "error-border" : ""}`}
+          placeholder="Name"
+          value={formData.name}
+          onChange={handleChange}
+        />
+        {errors.name && <p className="error-text">{errors.name}</p>}
+        
+        <input
+          type="number"
+          name="people"
+          className={`form-input ${errors.people ? "error-border" : ""}`}
+          placeholder="How many people"
+          value={formData.people}
+          onChange={handleChange}
+        />
+        {errors.people && <p className="error-text">{errors.people}</p>}
+        
+        <DatePicker
+          selected={selectedDateTime}
+          onChange={(date) => setSelectedDateTime(date)}
+          showTimeSelect
+          dateFormat="Pp"
+          filterTime={(time) => {
+            const hours = time.getHours();
+            return hours >= 10 && hours < 24;
+          }}
+          minDate={new Date()}
+          maxDate={new Date(new Date().setMonth(new Date().getMonth() + 1))}
+          placeholderText="Select date and time"
+          className={`form-input ${errors.date ? "error-border" : ""}`}
+        />
+        {errors.date && <p className="error-text">{errors.date}</p>}
+        
+        <textarea
+          name="message"
+          className={`form-input ${errors.message ? "error-border" : ""}`}
+          placeholder="Message / Special Requirements"
+          value={formData.message}
+          onChange={handleChange}
+          maxLength={250}
+        />
+        {errors.message && <p className="error-text">{errors.message}</p>}
```

Comment:
better use errors?.message to avoid errors

Reply:
fixed